css-layout-classes/styles.css

```diff
@@ -0,0 +1,35 @@
+#main-container {
+  padding: 0 100px;
+  font-family:
+    Arial,
+    Helvetica,
+    sans-serif;
+  font-size: large;
+}
+
+.main-head {
+  font-size: xx-large;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.wide-text {
```

Comment:
column-two-thirds would be a better semantic name for this class since that is the amount of space it is taking up.